build/yamls/base/conf/antrea-agent.conf

```diff
@@ -10,6 +10,11 @@ featureGates:
 # this flag will not take effect.
 #  EndpointSlice: false
 
+# Enables ServiceInternalTrafficPolicy in AntreaProxy. Don't enable this feature unless the
+# ServiceInternalTrafficPolicy feature gate of kube-apiserver is also enabled. If AntreaProxy is
+# not enabled, this flag will not take effect.
+#  ServiceInternalTrafficPolicy: false
```

Comment:
Is this worth a feature gate? I assume it doesn't affect the code a lot. And we didn't create a feature gate for NetworkPolicy EndPort, which is similar. Maybe do same thing here: if service's the policy is set to internal (meaning the feature is enabled on K8s side), implement it as it's desired. 

@jianjuns @antoninbas what you think?

Reply:
I do not like to add too many feature gate either. If the changes are not limited, probably not worth a feature gate.

Reply:
That's also my preferred solution. It seems that it is fine to call `svcInfo.NodeLocalInternal()` even if the K8s version doesn't support `ServiceInternalTrafficPolicy`.

https://github.com/kubernetes/kubernetes/blob/b0bc8adbc2178e15872f9ef040355c51c45d04bb/pkg/api/v1/service/util.go#L81-L86